configs/dh-defaults.prop

```diff
@@ -45,123 +14,41 @@ measurement.log_and_reset=false
 # TODO: all of the created threads are waiting on RPCs or locks.
 #
 NIO.driver.workTimeout=100
-NIO.driver.initialThreadCount=2
-NIO.driver.maxThreadCount=400
+NIO.driver.initialThreadCount=4
+NIO.driver.maxThreadCount=16
```

Comment:
Remind me, what happens at exhaustion? Do we throw exception, or schedule for next available thread.

Reply:
We shutdown the process.

Reply:
That said, we shouldn't be using this thread pool much anymore.

Reply:
This was included in grpc-api.prop to begin with because the server wouldnt start without it. Startup today without that:

```
Initiating shutdown due to: Uncaught exception in thread UpdateGraphProcessor.DEFAULT.refreshThread
io.deephaven.configuration.PropertyException: Missing property in config file: NIO.driver.maxThreadCount
        at io.deephaven.configuration.PropertyFile.getProperty(PropertyFile.java:115)
        at io.deephaven.configuration.PropertyFile.getInteger(PropertyFile.java:126)
        at io.deephaven.net.impl.nio.NIODriver.init(NIODriver.java:106)
        at io.deephaven.net.impl.nio.NIODriver.init(NIODriver.java:96)
        at io.deephaven.net.CommBase.getScheduler(CommBase.java:68)
        at io.deephaven.engine.updategraph.UpdateGraphProcessor.refreshTablesAndFlushNotifications(UpdateGraphProcessor.java:1457)
        at io.deephaven.engine.updategraph.UpdateGraphProcessor$1.run(UpdateGraphProcessor.java:208)

```